frontend/src/components/profile/settings.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+
+import {
+  withStyles,
+  Paper,
+  Grid,
+  Button
+} from '@material-ui/core'
+
+import { FormattedMessage, injectIntl } from 'react-intl'
+import DeleteUser from './settings/deleteUser'
+
+const styles = theme => ({
+  deleteButton: {
+    backgroundColor: '#F52A2A',
+    color: '#E5E5E5',
+    '&:hover': {
+      backgroundColor: '#F16060'
```

Comment:
@spikeboom we should use theme colors like:
https://github.com/worknenjoy/gitpay/search?q=theme.&unscoped_q=theme.

Please check the color associate with danger using Material-UI: 
https://material-ui.com/customization/color/

Reply:
I did not find the color palette for the danger type in the Material UI documentation. So I used the error type. Tell me if I'm right. For any changes I am ready for adjustments.